app/backend/src/clj/teet/meeting/meeting_commands.clj

```diff
@@ -434,3 +439,12 @@
           :participation/in "new-meeting"
           :participation/role (:db/ident role)
           :participation/participant (:db/id participant)})))})
+
+
+(defcommand :meeting/seen
+  {:doc "Mark meeting seen timestamp for user"
+   :payload {id :db/id}
+   :context {:keys [db user]}
+   :project-id (project-db/meeting-project-id db id)
+   :authorization {:project/read-info {:eid (project-db/meeting-project-id db id)}}
+   :transact [(entity-db/entity-seen-tx db user id)]})
```

Comment:
This reminded me that we discussed whether we should make it a macro expansion time fail if there's no schema for the payload.

Reply:
do you mean spec for payload?

Reply:
Yeah spec. :) Too many years working with Plumatic schema takes it's toll.

Reply:
I'll add inline spec for this, but idk if macro expansion time is actually good, as the spec might be registered later.